### PR TITLE
Add command to chaincli for setting up a bootstrap node

### PR DESCRIPTION
**Pull request by begmaroman**
```
chaincli bootstrap --ui-port <port> --p2pv2-port <port> <contract-address>
```

### CODE REVIEW

File: core/scripts/chaincli/handler/bootstrap.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/web"
+)
+
+const (
+	bootstrapJobSpec = `type = "bootstrap"
+schemaVersion = 1
+name = "ocr2keeper bootstrap node"
+externalJobID = "%s"
+id = "1"
```
Comment (infiloop2) [inline core/scripts/chaincli/handler/bootstrap.go:21]: should this always be id = 1, or can we leave it blank to get auto generated?

File: core/scripts/chaincli/handler/handler.go [hunk 2]
```diff
@@ -4,18 +4,41 @@ import (
...
 
+	"github.com/smartcontractkit/chainlink/core/cmd"
 	link "github.com/smartcontractkit/chainlink/core/gethwrappers/generated/link_token_interface"
+	"github.com/smartcontractkit/chainlink/core/logger"
 	"github.com/smartcontractkit/chainlink/core/scripts/chaincli/config"
+	"github.com/smartcontractkit/chainlink/core/sessions"
+)
+
+const (
+	defaultChainlinkNodeImage = "smartcontract/chainlink:1.5.1-root"
```
Comment (infiloop2) [inline core/scripts/chaincli/handler/handler.go:37]: why? this is very old

File: core/scripts/chaincli/handler/handler.go [hunk 3]
```diff
@@ -130,14 +153,225 @@ func (k *Keeper) sendEth(ctx context.Context, to common.Address, amount *big.Int
...
+		return "", nil, fmt.Errorf("failed to start DB container: %s", err)
+	}
+	log.Println("Postgres docker container successfully created and started: ", dbContainerResp.ID)
+
+	dbContainerInspect, err := dockerClient.ContainerInspect(ctx, dbContainerResp.ID)
+	if err != nil {
+		return "", nil, fmt.Errorf("failed to inspect Postgres container: %s", err)
+	}
+
+	time.Sleep(time.Second * 10)
```
Comment (infiloop2) [inline core/scripts/chaincli/handler/handler.go:216]: hacky, but might be fine :p

**Review by infiloop2 — COMMENTED** [review]
lgtm on a high level, left some minor comments

File: core/scripts/chaincli/handler/handler.go (hunk 3 above)
Comment (begmaroman) [reply core/scripts/chaincli/handler/handler.go:216]: This is not a new code as well. Just moved to another file.

File: core/scripts/chaincli/handler/handler.go (hunk 2 above)
Comment (begmaroman) [reply core/scripts/chaincli/handler/handler.go]: I just moved this version to this file :) Updated to the latest one

File: core/scripts/chaincli/handler/bootstrap.go (hunk 1 above)
Comment (begmaroman) [reply core/scripts/chaincli/handler/bootstrap.go]: Done

**Pull request merged**